Starting from version 2006.12.08, Gentoo installs live555 as a system
library properly. For policy we also build a shared version of it that
uses PIC.
For this reason, we can reasonably force VLC to create a Plugin for
live555 rather than a builtin.
===================================================================
Index: vlc/configure.ac
===================================================================
--- vlc.orig/configure.ac
+++ vlc/configure.ac
@@ -1920,7 +1920,7 @@ if test "${enable_live555}" != "no"; the
           # We only have -lliveMedia, do builtins
           VLC_ADD_PLUGIN([live555])
           VLC_ADD_LIBS([live555], [-lliveMedia -lgroupsock -lBasicUsageEnvironment -lUsageEnvironment])
-        ])
+        ], ,[-lgroupsock -lBasicUsageEnvironment -lUsageEnvironment])
       ])
       if test "${SYS}" = "mingw32"; then
         # add ws2_32 for closesocket, select, recv
